frontend/src/components/NotificationsBar/NotificationsBarContainer.tsx

```diff
@@ -11,38 +12,40 @@ interface Props {
   notifications?: NotificationType[];
 }
 
-const NotificationsBarContainer: React.FC<Props> = ({
+const NotificationsBarContainer: FC<Props> = ({
   isMobileMenuOpen,
   toggleMobileMenu,
   notifications,
 }) => {
   const [isNotificationsOpen, setNotificationsOpen] = useState<boolean>(false);
-  const [isContainsUnread, setUnread] = useState<boolean>(false);
+
+  const { mutateAsync: readNotificationsMutation } = useReadNotifications();
```

Comment:
for what you using mutateAsync? Its returns Promise. You dont working with try catch here. Use mutate insteade mutateAsync. https://react-query.tanstack.com/guides/mutations